src/transformers/tokenization_utils_base.py

```diff
@@ -715,15 +715,14 @@ def convert_to_tensors(
 
                     self[key] = tensor
             except:  # noqa E722
-                if key == "overflowing_tokens":
+                if key == "overflowing_tokens" or key == "input_ids" or key == "attention_mask":
```

Comment:
I'm not sure if there are more keys to take take into account for this issue

Reply:
I think there was a miscommunication there.

In the OP you suggested that there could be an error of passing wrongly shaped inputs - at least that's how I understood it. And I proposed that perhaps it'd be better to check if inputs are misformatted and assert if that's the case. But your newly proposed change is something totally different. So I think I lost you here.

Perhaps you could show an example of wrongly shaped inputs and then the corrected one?